index.js

```diff
@@ -58,16 +63,22 @@ class OmronSensorService extends EventEmitter {
         this.handleError = this.handleError.bind(this);
         this.isOmronEvent = this.isOmronEvent.bind(this);
         this.passWhitelistCheck = this.passWhitelistCheck.bind(this);
+        this.isInvalidEvent = this.isInvalidEvent.bind(this);
+        this.isCooledDown = this.isCooledDown.bind(this);
+        this.isDuplicateEvent = this.isDuplicateEvent.bind(this);
+
```

Comment:
Might need
`this.passEventChecks = this.passEventChecks.bind(this);`
But if it works it works 🤷